store/store.go

```diff
@@ -283,8 +301,9 @@ func (bs *BlockStore) PruneBlocks(height int64) (uint64, error) {
 
 	pruned := uint64(0)
 	batch := bs.db.NewBatch()
+	batchTxs := bs.db.NewBatch()
```

Comment:
Why do you create a different batch. You should use the same batch in order for the transaction to be atomic

Reply:
first batch is for the blocks second one is for txs